### ISSUE TITLE
0.8.0 aggregation fails (UI hangs) on some CDs

### ISSUE THREAD

**Issue by mzipay** [mzipay/FLACManager#4]
Looks like an issue with aggregation of custom metadata:

```
$ ./flacmanager.py
Exception in thread Thread-48:
Traceback (most recent call last):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "./flacmanager.py", line 5774, in run
    self.aggregate()
  File "./flacmanager.py", line 5822, in aggregate
    collector.metadata["__custom"], self.metadata["__custom"])
KeyError: '__custom'
```

**Comment by mzipay** [comment]
This is an issue related to the 0.8.0 change in structure for persisted metadata, I think. I originally ripped the CD that failed some time ago (prior to the introduction of custom metadata in 0.8.0).

**Comment by mzipay** [comment]
The problem is `MetadataPersistence.__convert_restored_metadata`, which doesn't add the "__custom" key to the disc/tracks metadata for older (pre-0.8.0) persisted metadata.

Fixed in 0.8.1-beta.
